components/BrandingContent/ColorSection/ColorBlock/ColorBlock.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+ColorBlock.propTypes = {
+  colorName: PropTypes.string.isRequired,
+  colorHex: PropTypes.string.isRequired,
+};
+
+function ColorBlock({ colorHex, colorName }) {
+  const colorBox = {
+    minWidth: '90%',
+    minHeight: '200px',
+    backgroundColor: colorHex,
```

Comment:
I prefer to not inline styles when I can. It's needed for the colorHex, but can you use CSS Modules to integrate the other constant CSS values?